translations/ai_rules.tex/de.po

```diff
@@ -172,36 +232,50 @@ msgid ""
 "If an AI Hero is instructed to draw a card, they will draw and resolve \\textbf{another card} from the AI Deck.\n"
 "\n"
 msgstr ""
+"Wenn ein Bot-Held eine Einheit \\textbf{aktiviert}, ziehe eine Bot-Karte\\index{Bot-Karte} und befolge ihre Anweisungen, bevor sich die Einheit bewegt und/oder angreift.\n"
+"Ziehe keine Karten mehr, wenn das Bot-Deck während des Kampfes aufgebraucht ist.\n"
+"Der Effekt jeder Bot-Karte hängt vom \\pagelink{Difficulty}{Schwierigkeitsgrad} des Spiels ab.\n"
+"Bot-Machtkarten \\svg{might} werden an die Einheit angehängt, bis der/die erste entsprechende Angriff/Verteidigung erfolgt ist.\n"
+"Das Bot-Zauberdeck wird verwendet, wenn eine Bot-Zauberkarte \\svg{magic} gezogen wird.\n"
+"Wenn ein Bot-Held eine Karte ziehen soll, zieht er eine weitere Karte aus dem Bot-Deck und handelt sie ab.\n"
+"\n"
 
 #. type: multicols
-#: sections/ai_rules.tex:81
+#: sections/ai_rules.tex:80
 #, no-wrap
 msgid ""
 "\\subsection*{Combat against AI}\n"
 "\n"
-msgstr ""
+msgstr "\\subsection*{Kämpfe gegen Bots}\n"
 
 #. type: multicols
-#: sections/ai_rules.tex:84
+#: sections/ai_rules.tex:83
 #, no-wrap
 msgid ""
 "\\pagetarget{AI Units}{These rules} apply during Combat in \\textbf{Solo}\\index{Solo Mode} and \\textbf{Cooperative} Scenarios.\n"
 "When Neutral enemies or AI Heroes activate a unit, they follow a set of automatic instructions:\\par\n"
 "\n"
 msgstr ""
+"\\pagetarget{AI Units}{Diese Regeln} gelten während des Kampfes in \\textbf{Solo-}\\index{Solomodus} und \\textbf{kooperativen} Szenarien.\n"
+"Wenn neutrale Feinde oder Bot-Helden eine Einheit aktivieren, folgen sie einer Reihe von automatischen Anweisungen:\\par\n"
+"\n"
 
 #. type: multicols
-#: sections/ai_rules.tex:88
+#: sections/ai_rules.tex:87
 #, no-wrap
 msgid ""
 "\\begin{itemize}\n"
 "  \\item Enemy Ground \\svg{unit_ground} and Flying \\svg{unit_flying} Units prioritize attacking Units of the \\textbf{same} tier.\n"
 "  If this is impossible, they attack the Unit of a lower tier (in tier \\textbf{descending} order, down to bronze), and if that is also impossible, they attack the Unit of a higher tier (in tier \\textbf{ascending} order).\\par\n"
 "\n"
 msgstr ""
+"\\begin{itemize}\n"
```

Comment:
```suggestion
"\\begin{itemize}[leftmargin=15pt]\n"
```

Reply:
This, plus get rid of this orphaned "ab", and everything fits nicely.
![image](https://github.com/user-attachments/assets/26f4c181-4b4d-4c38-b618-b4f88d9214cb)